.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+
+version: 2
+
+defaults: &defaults
+  docker:
+    - image: circleci/ruby:2.3
+  working_directory: ~/www.goodcity
+
+jobs: 
+  checkout_code:
+    <<: *defaults
+    steps:
+      - checkout
+      - save_cache:
+          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+          paths:
+            - ~/www.goodcity
+  test:
```

Comment:
Given there are no tests for this project, I would delete this step entirely

Reply:
Yup.. I was also thinking why it was there in circle file. I will remove it. :)